pkg/storage/stores/shipper/bloomshipper/client.go

```diff
@@ -102,84 +100,40 @@ type Client interface {
 	Stop()
 }
 
-// todo add logger
-func NewBloomClient(periodicConfigs []config.PeriodConfig, storageConfig storage.Config, clientMetrics storage.ClientMetrics) (*BloomClient, error) {
-	periodicObjectClients := make(map[config.DayTime]client.ObjectClient)
-	for _, periodicConfig := range periodicConfigs {
-		objectClient, err := storage.NewObjectClient(periodicConfig.ObjectType, storageConfig, clientMetrics)
-		if err != nil {
-			return nil, fmt.Errorf("error creating object client '%s': %w", periodicConfig.ObjectType, err)
-		}
-		periodicObjectClients[periodicConfig.From] = objectClient
-	}
-	return &BloomClient{
-		periodicConfigs:       periodicConfigs,
-		storageConfig:         storageConfig,
-		periodicObjectClients: periodicObjectClients,
-	}, nil
-}
+var _ Client = &BloomClient{}
```

Comment:
Nit: this isn't immediately clear to everyone that we're using this as a compiler check.

Reply:
Will add a comment.